pom.xml

```
@@ -40,6 +40,12 @@
       <artifactId>okhttp-api</artifactId>
       <version>3.14.9</version>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>jjwt-api</artifactId>
+      <version>0.11.2-4.ba1e9390e5ff</version>
```

from https://github.com/jenkinsci/jjwt-api-plugin/pull/1
```suggestion
      <version>0.11.2-5.143e44951c52</version>
```